vision/product_search/create_product.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+// [START imports]
```

Comment:
Might need a more specific region tag?

Reply:
updated to include the imports within the ample region tag.